### DISCUSSION TITLE
AppBar and Card widget

### DISCUSSION THREAD

**Discussion by Yuerchu** [zauberzeug/nicegui#3357] [Ideas / Feature Requests]
English from machine translate

I just came across this component library today and it blew me away. It's a nearly perfect one 'cause it's super easy for beginners like me to handle, and it's quite similar to Material Design :)

It would be much better if some libraries were added, like [AppBar](https://www.mdui.org/en/docs/2/components/top-app-bar), [Card](https://www.mdui.org/en/docs/2/components/card), and so on.

今天刚刚看到这个组件库，惊艳到我了，这是一个无限趋近于完美的库，因为它能让我这个初学者都轻易上手，而且很像Material Design :)

如果能加上一些库就更完美了，比如[AppBar](https://www.mdui.org/zh-cn/docs/2/components/top-app-bar)、[Card](https://www.mdui.org/zh-cn/docs/2/components/card)等等。

**Comment by python-and-novella** [comment]
```python3
from nicegui import ui
with ui.element('q-bar').classes('w-full'):
    ui.space()
    ui.icon('bluetooth')
    ui.icon('signal_cellular_4_bar')
    ui.icon('signal_wifi_4_bar')
    ui.icon('battery_full')
with ui.card().classes('w-96 h-96'):
    ui.label('Test Text')
ui.run(native=True,reload=True)
```
![image](https://github.com/user-attachments/assets/97395fd1-f2ca-4350-ae93-750b2de003d2)

**Reply by python-and-novella** [reply]
https://quasar.dev/vue-components/toolbar
![image](https://github.com/user-attachments/assets/1f72e9d2-159f-49ee-9430-393c24e8a903)

**Reply by Yuerchu** [reply]
Thanks^_^！ 非常感谢！
